templates/inspec/tests/integration/build/gcp-mm.tf

```diff
@@ -838,3 +838,21 @@ resource "google_compute_node_group" "inspec-node-group" {
   size = var.node_group["size"]
   node_template = "${google_compute_node_template.inspec-template.self_link}"
 }
+
+resource "google_spanner_instance" "spanner_instance" {
+  project      = "${var.gcp_project_id}"
+  config       = "${var.spannerinstance["config"]}"
+  name         = "${var.spannerinstance["name"]}"
+  display_name = "${var.spannerinstance["display_name"]}"
+  node_count   = "${var.spannerinstance["node_count"]}"
+  labels = {
+    "${var.spannerinstance["label_key"]}" = "${var.spannerinstance["label_value"]}"
+  }
+}
+
+resource "google_spanner_instance" "database" {
```

Comment:
This should probably be a `google_spanner_database` instead of a `google_spanner_instance`.

Are you able to run these tests locally after running magic modules? You should be able to generate the inspec-gcp changes and try to run the integration tests in inspec-gcp yourself. 

This page has information on running the integration tests: https://github.com/inspec/inspec-gcp#test-inspec-gcp-resources

Reply:
Correct this. Sorry about another copy paste error.. strange my local test didnt caught this.